src/Context/index.jsx

```diff
@@ -34,9 +34,10 @@ export const ContextWrapper = props => {
     try {
       var messages = "Event successfully created";
       const response = await Axios.post("/trainings", data);
-      Notification(messages);
+      Notification(messages, "success");
     } catch (error) {
-      console.log(error);
+      console.log(21111, error);
```

Comment:
remove log

Reply:
thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0